### DISCUSSION TITLE
Is SSH.NET Vulnerable to regreSSHion Vulnerability?

### DISCUSSION THREAD

**Discussion by motisoft** [sshnet/SSH.NET#1436] [Q&A] [answered]
I saw that there is at least one vulnerability named "**OpenSSH Remote Unauthenticated Code Execution Vulnerability (regreSSHion)**" that is OS agnostic.

**Comment by Rob-Hague** [comment, marked as answer]
No, this is an SSH client. This is not OpenSSH Server
